### PR TITLE
Upgrade docs fixes

### PR DESCRIPTION
**Pull request by lmm**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit dbbb8c1956fa168c1e9b390f2407f919c9c2a515

https://deploy-preview-3516--calico.netlify.app

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit a9ceebfe966e4e6ffb2c803107028b5592dcb438

https://deploy-preview-3516--calico.netlify.app

File: _includes/content/hostendpoints-upgrade.md [hunk 1]
```diff
@@ -13,16 +13,49 @@ is to **drop traffic**. This default behavior is consistent with "named" host en
...
+
+```bash
+calicoctl get hep -owide | grep '*' | awk '{print $1}'
+```
+
+With the names of the all-interfaces host endpoints, we can label each host endpoint with a new label (for example, **host-endpoint-upgrade: ""**):
+
+```bash
+calicoctl get hep -owide | grep '*' | awk '{print $1}' \
+  | xargs -I {} kubectl exec -i -n kube-system calicoctl /calicoctl label hostendpoint {} host-endpoint-upgrade=
```
Comment (lwr20) [inline _includes/content/hostendpoints-upgrade.md:27]:
```suggestion
calicoctl get hep -owide | grep '*' | awk '{print $1}' \
  | xargs -I {} kubectl exec -i -n kube-system calicoctl -- /calicoctl label hostendpoint {} host-endpoint-upgrade=
```

File: _includes/content/auto-hostendpoints-migrate.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+2. Enable auto host endpoints by following the [enable automatic host endpoints how-to guide]({{ site.baseurl }}/security/kubernetes-nodes#enable-automatic-host-endpoints).
+   Note that automatic host endpoints are created with a profile attached that allows all traffic in the absence of network policy.
+
+   ```bash
+   calicoctl patch kubecontrollersconfiguration default --patch='{"spec": {"controllers": {"node": {"hostEndpoint": {"autoCreate": "Enabled"}}}}}'
+   ```
+3. Delete old all-interfaces host endpoints. You can distinguish host endpoints managed by {{site.prodname}} from others in several ways. First, automatic host endpoints
+   have the label **projectcalico.org/created-by: calico-kube-controllers**. Secondly, automatic host endpoints' name have the suffix **-auto-hep**.
+
```
Comment (lwr20) [inline _includes/content/auto-hostendpoints-migrate.md:25]: Does it need a vague step 4 to reinstate any old policies?

Comment (lmm) [reply _includes/content/auto-hostendpoints-migrate.md:25]: I added a step to the upgrade steps, just before the declaration of upgrade complete: 170d9e50e3f401b9a1ae1e1696b63533b50d6f7c

File: maintenance/openstack-upgrade.md [hunk 3]
```diff
@@ -139,4 +139,7 @@ varies by Linux distribution.
 
 1. Remove any existing `calicoctl` instances and [install the new `calicoctl`](../getting-started/clis/calicoctl/install).
 
+1. If you were upgrading from a version of Calico prior to v3.14 and followed the pre-upgrade steps for host endpoints above, review traffic logs from the temporary policy,
```
Comment (lwr20) [inline maintenance/openstack-upgrade.md:142]: Er - this host endpoints thing - does it apply to openstack?

File: maintenance/openshift-upgrade.md [hunk 4]
```diff
@@ -31,3 +31,8 @@ You can now monitor the upgrade progress with the following command:
 ```bash
 watch oc get tigerastatus
 ```
+
+If you were upgrading from a version of Calico prior to v3.14 and followed the pre-upgrade steps for host endpoints above, review traffic logs from the temporary policy,
+add any global network policies needed to whitelist traffic, and delete the temporary network policy **allow-all-upgrade*.
+
+{% include content/auto-hostendpoints-migrate.md orch="OpenShift" %}
```
Comment (lwr20) [inline maintenance/openshift-upgrade.md:38]: Is there any sane way to have the Openshift doc use `oc` instead of `kubectl`?

Comment (lmm) [reply maintenance/openshift-upgrade.md:38]: c3dd0d7 and 0c919e5

File: maintenance/openstack-upgrade.md (hunk 3 above)
Comment (lmm) [reply maintenance/openstack-upgrade.md]: c3dd0d7

**Pull request merged**